test/bisect.test.js

```diff
@@ -0,0 +1,16 @@
+const chai = require('chai')
+  .use(require('dirty-chai'))
+const { expect } = chai
+const { bisect } = require('../src/bisect')
+
+module.exports = () => {
+  describe('bisect tests', () => {
+    it('must find the nearest index', async () => {
```

Comment:
since we moved bisect implementation here please add other test cases similar to https://github.com/bitfinexcom/hypercore-bisect/blob/master/test/basic.js